src/components/ArticlesList.jsx

```diff
@@ -1,9 +1,24 @@
-const ArticlesList = () => {
+import { useEffect, useState } from "react";
+import { fetchArticles } from "./Api";
 
+const ArticlesList = () => {
+  const [articlesList, setArticlesList] = useState([]);
 
-    return (
-        <h2>Articles</h2>
-    )
-}
+  useEffect(() => {
+    fetchArticles().then((data) => {
```

Comment:
Could we add some loading state to let the user know that something is happening? Especially since the server is quite slow at first

Reply:
I was thinking to do this but wasn't sure it was the right time to implement it, thanks will do that now